package.json

```diff
@@ -102,5 +104,5 @@
       "last 1 safari version"
     ]
   },
-  "homepage": "/dashboard/"
+  "homepage": "/polkadot-staking-dashboard/"
```

Comment:
@rossbulat this breaks the deploy,
could you try: (detail here: https://stackoverflow.com/a/44048328)
`"homepage": ".",`

if that is not good enough you can pass like this:
https://stackoverflow.com/a/61070071

Reply:
Thanks for identifying. This should just be "/dashboard", it was erroneously changed to the github deployment URI somewhere along the line.

Fixed in the latest commit.